internal/core/uiapi/api.go

```diff
@@ -0,0 +1,702 @@
+package uiapi
+
+import (
+	"bytes"
+	"crypto/tls"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"go.uber.org/zap"
+
+	"github.com/playmixer/secret-keeper/internal/adapter/api/rest"
+	"github.com/playmixer/secret-keeper/internal/adapter/models"
+)
+
+var (
+	errMessageFailedReadBody   = "failed read body"
+	errMessageFailedUnmarshal  = "failed unmarshal"
+	errMessageFailedCreateCard = "failed create card"
+	errMessageFailedRequest    = "failed request"
+	errMessageFailedCloseBody  = "failed close body response"
+
+	formatStringError = "%s: %w"
+)
+
+func (k *keepClient) EventAuthorization(login, password string) error {
+	req := tSignInRequest{
+		Login:    login,
+		Password: password,
+	}
+
+	bReq, err := json.Marshal(req)
+	if err != nil {
+		k.log.Error("failed marshal request", zap.Error(err))
+		return fmt.Errorf("failed marshal request: %w", err)
+	}
+
+	r, err := k.newRequest(http.MethodPost, k.apiURL+"/api/v0/auth/login", &bReq)
+	if err != nil {
+		k.log.Error("failed create request", zap.Error(err))
+		return fmt.Errorf("failed create request: %w", err)
+	}
+
+	res, err := io.ReadAll(r.Body)
+	if err != nil {
+		k.log.Error(errMessageFailedReadBody, zap.Error(err))
+		return fmt.Errorf(formatStringError, errMessageFailedReadBody, err)
+	}
+	defer func() {
+		err := r.Body.Close()
+		if err != nil {
+			k.log.Error("failed close body", zap.Error(err))
+		}
+	}()
+
+	if r.StatusCode == http.StatusUnauthorized {
+		return errors.New("логии или пароль не верный")
```

Comment:
```suggestion
		return errors.New("логин или пароль не верный")
```

А что значит "неверный" логин ?